app/controllers/api/v1/students_controller.rb

```diff
@@ -11,6 +11,22 @@ def show
     	# @skills = @student.skills
   end
 
+  def create
+    @student = Student.create({
+      :first_name => params[:first_name],
+      :last_name => params[:last_name],
+      :email => params[:email],
+      :phone_number => params[:phone_number],
+      :github => params[:github],
+      :blog => params[:blog],
+      :twitter => params[:twitter],
+      :linkedin => params[:linkedin]
+    })
+    # redirect_to api_v1_student_path(@student.id)
```

Comment:
I think we need to add something for the sad path such as this..

if @student.save
    else
      render json: { errors: @spot.errors.full_messages }, status: 422
    end